<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(195, 200, 206);</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_es">
    <property name="title">
     <string>Ações</string>
    </property>
    <addaction name="actionVer_N_meros"/>
    <addaction name="actionInserir"/>
    <addaction name="actionSuporte"/>
    <addaction name="actionTransferencia"/>
   </widget>
   <addaction name="menuA_es"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionVer_N_meros">
   <property name="text">
    <string>Ver Números</string>
   </property>
  </action>
  <action name="actionInserir">
   <property name="text">
    <string>Inserir</string>
   </property>
  </action>
  <action name="actionTransferir">
   <property name="text">
    <string>Transferir</string>
   </property>
  </action>
  <action name="actionSuporte">
   <property name="text">
    <string>Suporte</string>
   </property>
  </action>
  <action name="actionTransferencia">
   <property name="text">
    <string>Extração</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
